### DISCUSSION TITLE
Can't deploy to edge, but can to serverless

### DISCUSSION THREAD

**Discussion by kelbyfaessler** [vercel/community#1843] [Help] [answered]
### Summary

When I try to deploy my sveltekit app to the edge runtime, I get the following error even though I don't think I'm using prisma in client code?

`Error: PrismaClient is unable to be run in the browser.`

And when I deploy to the `nodejs18.x` serverless runtime, I don't get that error.

Anyone know what might be the issue?

**Comment by swarnava** [comment]
Hi,

Are you following this instruction: https://www.prisma.io/blog/database-access-on-the-edge-8F0t1s1BqOJE ?

**Reply by kelbyfaessler** [reply, marked as answer]
Thanks, followed that guide, got stuck, and eventually had to set the following in the scripts section of my package.json
```
        "postinstall": "prisma generate --data-proxy"
```
